src/app/personal-details/page.tsx

```diff
@@ -0,0 +1,260 @@
+"use client";
+
+import React, { useState } from "react";
+import {
+	Box,
+	Button,
+	TextField,
+	Container,
+	Typography,
+	IconButton,
+} from "@mui/material";
+import AddPhotoAlternateIcon from "@mui/icons-material/AddPhotoAlternate";
+
+const PersonalDetailsScreen: React.FC = () => {
+	const [name, setName] = useState("");
+	const [username, setUsername] = useState("");
+	const [email, setEmail] = useState("");
+	const [jobTitle, setJobTitle] = useState("");
+	const [bio, setBio] = useState("");
+	const [linkedin, setLinkedin] = useState("");
+	const [aboutMe, setAboutMe] = useState("");
+
+	const handleNext = () => {
+		// Form submission logic here
+	};
+
+	return (
+		<Box
+			sx={{
+				minHeight: "100vh",
+				display: "flex",
+				justifyContent: "center",
+				alignItems: "center",
+				background: "linear-gradient(135deg, #F7F7F8, #E3E7FF, #DCE0FF)",
+			}}
+		>
+			<Container
+				maxWidth="md"
+				sx={{
+					backgroundColor: "#FFFFFF",
+					padding: "48px",
+					borderRadius: "16px",
+					border: "1px solid #D6D6E7",
+					boxShadow: "0px 4px 12px rgba(0, 0, 0, 0.1)",
+				}}
+			>
+				{/* Header */}
+				<Typography
+					variant="h5"
+					component="h1"
+					sx={{
+						textAlign: "center",
+						color: "#000000",
+						fontWeight: 600,
+						fontSize: "32px",
+						lineHeight: "40.32px",
+						marginBottom: "24px",
+						fontFamily: "'Outfit', sans-serif",
+					}}
+				>
+					Tell Us About Yourself
+				</Typography>
+				<Typography
+					variant="body2"
+					sx={{
+						textAlign: "center",
+						color: "#000000",
+						marginBottom: "32px",
+						fontSize: "24px",
+						lineHeight: "30.24px",
+						fontFamily: "'Outfit', sans-serif",
+						fontWeight: 200,
+					}}
+				>
+					This helps other founders understand who you are when reviewing your
+					ideas.
```

Comment:
I think the text weight is to much. Maybe something less heavy. Something to be discussed with the designers.

Reply:
got it! let's leave like this for now and discuss with the designers.